_translator-files/po/fr/Tips-Tricks-More.po

```diff
@@ -82,8 +83,23 @@ msgstr "### Découvrir les répétitions de groupes à distance"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:23
-msgid "Jamulus user [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) has compiled a massive amount of information relating to [Remote Band Rehearsals](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc), which covers topics such as hardware and software configuration including examples and advice for newcomers to the field. He also includes a section on Jamulus in comparison to other solutions."
-msgstr "L'utilisateur de Jamulus [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) a rassemblé une grande quantité d'informations sur les [Répétitions à distance de groupes musicaux](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc), qui couvre des sujets tels que la configuration matérielle et logicielle, y compris des exemples et des conseils pour les nouveaux venus dans le domaine. Il inclut également une section sur Jamulus en comparaison avec d'autres solutions."
+msgid ""
+"Jamulus user [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) "
+"has compiled a massive amount of information relating to [Remote Band "
+"Rehearsals](https://docs.google.com/document/"
+"d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc), which covers "
+"topics such as hardware and software configuration including examples and "
+"advice for newcomers to the field. He also includes a section on Jamulus in "
+"comparison to other solutions."
+msgstr ""
+"L'utilisateur de Jamulus [Chris Rimple](https://sourceforge.net/u/"
+"chrisrimple/profile/) a rassemblé une grande quantité d'informations sur les "
+"[Répétitions à distance de groupes musicaux](https://docs.google.com/"
+"document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc), qui "
+"couvre des sujets tels que la configuration matérielle et logicielle, y "
+"compris des exemples et des conseils pour les nouveaux venus dans le "
+"domaine. Il inclut également une section sur Jamulus en comparaison avec "
+"d'autres solutions."
```

Comment:
I wonder why Weblate decided to wrap all the lines in this doc (I checked that the add-on preventing this is installed)?

Reply:
It does make reviewing on Github nigh on impossible...

Reply:
Do you mean it's harder when it's wrapped or when it's not?

Reply:
I mean I find it hard because it has been wrapped.  Seeing what exactly changed isn't very easy due to the wrapping obscuring everything.

So long as the wrapping is stable between updates, I don't mind one way or the other -- but I don't want it changing it ad hoc.